crates/store/src/state_db.rs

```diff
@@ -196,15 +327,20 @@ impl<'a> CodeStore for StateTree<'a> {
             None => None,
         }
     }
+
     fn insert_data(&mut self, data_hash: H256, code: Bytes) {
         self.db
             .insert_raw(COLUMN_DATA, data_hash.as_slice(), &code)
             .expect("insert data");
     }
+
     fn get_data(&self, data_hash: &H256) -> Option<Bytes> {
         match self.db.get(COLUMN_DATA, data_hash.as_slice()) {
             Some(slice) => Some(Bytes::from(slice.to_vec())),
             None => None,
         }
     }
 }
+
+#[cfg(test)]
+mod tests;
```

Comment:
We should put tests under `crates/store/src/tests` and define the tests module in the `crates/store/src/lib.rs`.

Reply:
This solution is ok and the file structure seems more clean. As the tests is defined as a module, which and the target module are at the same level, some build-in unit test functions like testing private are not be valid any more. Please note.